sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -305,7 +313,7 @@ def __init__(
         self._compute = ComputeOperations(
             self._operation_scope,
             self._operation_config,
-            self._service_client_10_2022_preview,
+            self._service_client_12_2022_preview,
```

Comment:
why do we need to change compute operations?

Reply:
This change comes from https://github.com/Azure/azure-sdk-for-python/pull/27924/files. Compute operations related lines are changed in this pr.

Reply:
I know, but we would want to avoid those in our PR.